recipes/server_conf.rb

```diff
@@ -22,7 +22,7 @@
   owner "postgres"
   group "postgres"
   mode 0600
-  notifies change_notify, 'service[postgresql]', :delayed
+  notifies change_notify, 'service[postgresql]', :immediately
```

Comment:
sure thing - please let me know if you need any further explanations, or if you have any doubts...  ; )